src/features/eye/eye_analyzer.h

```diff
@@ -0,0 +1,105 @@
+#ifndef _READMINDS_FEATURES_EYE_EYE_ANALYZER_H_
+#define _READMINDS_FEATURES_EYE_EYE_ANALYZER_H_
+
+#include "src/features/generic_analyzer.h"
+
+#include "mediapipe/framework/formats/landmark.pb.h"
+#include "mediapipe/framework/port/opencv_imgproc_inc.h"
+
+const int EYE_LEFT_INNER_LMARKS[] = {
+    7, 163, 144, 145, 153, 130, 154,
+    133, 173, 157, 158, 159, 160, 161,
+    190, 246, 243,
+};
+
+const int EYE_RIGHT_INNER_LMARKS[] =  {
+    249, 263, 362, 382, 381, 380, 374,
+    373, 388, 387, 386, 385, 384, 390,
+    398, 466, 463
+};
+
+const int EYE_BROW_LEFT_UPPER[] = {
+    70, 63, 105, 66, 107
+};
+
+const int EYE_BROW_RIGHT_UPPER[] = {
+    300, 293, 334, 296, 336
+};
+
+// lower eyebrow landmarks are not
+// being used, added for future purposes
+// (landmarks slightly below the previous ones)
+const int EYE_BROW_LEFT_LOWER[] = {
+    46, 53, 52, 65, 55
+};
+
+const int EYE_BROW_RIGHT_LOWER[] = {
+    300, 283, 282, 295, 285
+};
+
+/*
+    Class that computes eye related features,
+    e.g. eye area (F3) and eyebrow activity (F4).
+*/
```

Comment:
About the comment style (`/* */` or `//`), [google suggests](https://google.github.io/styleguide/cppguide.html#Comment_Style) using only one kind, and be consistent with it for the rest of the code. Since we'll have comments in the middle of code (in `.cc` files) I think using `//` would look prettier (also, typing `Ctrl+/` adds this type of comment :)). But any of the types would be good.

Reply:
I agree, `//` looks way better